src/modules/sway/workspaces.cpp

```diff
@@ -279,9 +284,24 @@ Gtk::Button &Workspaces::addButton(const Json::Value &node) {
 }
 
 std::string Workspaces::getIcon(const std::string &name, const Json::Value &node) {
-  std::vector<std::string> keys = {"urgent", "focused", name, "visible", "default"};
```

Comment:
Normal to remove visible?

Reply:
There is no documentation about that key neather in .scd or wiki. So it's literally undocumented option.
And it seems to overlap with `default` key.